skyportal/tests/api/test_catalog_services.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
+import pytest
+import time
+import uuid
+
+from skyportal.tests import api
+
+
+@pytest.mark.flaky(reruns=2)
+def test_swift_lsxps(super_admin_token):
+
+    name = str(uuid.uuid4())
+    status, data = api(
+        'POST',
+        'telescope',
+        data={
+            'name': name,
+            'nickname': name,
+            'lat': 0.0,
+            'lon': 0.0,
+            'elevation': 0.0,
+            'diameter': 10.0,
+        },
+        token=super_admin_token,
+    )
+    assert status == 200
+    assert data['status'] == 'success'
+    telescope_id = data['data']['id']
+
+    instrument_name = str(uuid.uuid4())
+    status, data = api(
+        'POST',
+        'instrument',
+        data={
+            'name': instrument_name,
+            'type': 'imager',
+            'band': 'NIR',
+            'filters': ['swiftxrt'],
+            'telescope_id': telescope_id,
+        },
+        token=super_admin_token,
+    )
+    assert status == 200
+    assert data['status'] == 'success'
+
+    status, data = api(
+        "POST",
+        "catalogs/swift_lsxps",
+        data={'telescope_name': name},
+        token=super_admin_token,
+    )
```

Comment:
Does this just get ALL the transients in that database?

Reply:
All 6 of them ;)